cmd/xmtpd-e2e/main.go

```diff
@@ -41,6 +41,20 @@ func main() {
 
 	apiURL := envVar("XMTPD_E2E_API_URL", remoteAPIURLByEnv[networkEnv])
 
+	walletKeyHex := envVar("XMTPD_E2E_AUTH_WALLET_KEY", "")
+	var walletKey *ecdsa.PrivateKey
+	if walletKeyHex != "" {
+		walletKey, err = crypto.HexToECDSA(walletKeyHex)
+		requireNoError(err)
+	}
+
+	installationKeyHex := envVar("XMTPD_E2E_AUTH_INSTALLATION_KEY", "")
```

Comment:
Installation is really a V3 term. This would be the identity key for v1/v2

Reply:
Will we be able to reuse these wallet/installation keys for the v3 auth identity too (as in will we need to pass in separate keys here for that or reuse these)?

Reply:
No possibility for re-use of the identity keys/installation keys. Totally different curves and signature schemes. Wallets obviously remain the same.